Automattic/wp-calypso
# NewsletterInto: fix backgrounds

As reported [here](https://github.com/Automattic/wp-calypso/issues/75684) the pattern used for background in the intro step was not symmetrical.

Related to #75684

## Proposed Changes
- Fix and adjust the images on desktop and mobile

## Testing Instructions
- Visit the newsletter intro step using the PR link
- Check if the alignment is fixed
